src/handlers/graphical.rs

```diff
@@ -688,6 +697,30 @@ impl GraphicalReportHandler {
         Ok(())
     }
 
+    fn render_highlighted_line_text(
+        &self,
+        f: &mut impl fmt::Write,
+        styled_text: &[Styled<&str>],
+    ) -> fmt::Result {
+        for styled in styled_text {
+            let styled_text = styled.to_string();
+            for (c, width) in styled_text
+                .chars()
+                .zip(self.line_visual_char_width(&styled_text))
```

Comment:
Don't zip the same value in a loop.
Create a single variable before loop, instead